packages/esm-patient-tests-app/src/lab-orders/lab-order-basket-panel/lab-order-basket-item-tile.component.tsx

```diff
@@ -24,37 +24,51 @@ export function LabOrderBasketItemTile({ orderBasketItem, onItemClick, onRemoveC
   // Hence, we manually prevent the handleClick callback from being invoked as soon as the button is pressed once.
   const shouldOnClickBeCalled = useRef(true);
 
+  const additionalInfoSlotState = useMemo(
+    () => ({
+      orderItemUuid: orderBasketItem.testType.conceptUuid,
+    }),
+    [orderBasketItem],
+  );
+
   const labTile = (
-    <div className={styles.orderBasketItemTile}>
-      <div className={styles.clipTextWithEllipsis}>
-        <OrderActionLabel orderBasketItem={orderBasketItem} />
-        <br />
-        <span className={styles.name}>{orderBasketItem.testType?.label}</span>
-        <span className={styles.label01}>
-          {!!orderBasketItem.orderError && (
-            <>
-              <br />
-              <span className={styles.orderErrorText}>
-                <WarningIcon size={16} />
-                &nbsp;
-                <span className={styles.label01}>{t('error', 'Error').toUpperCase()}</span> &nbsp;
-                {orderBasketItem.orderError.responseBody?.error?.message ?? orderBasketItem.orderError.message}
-              </span>
-            </>
-          )}
-        </span>
+    <div>
+      <div className={styles.orderBasketItemTile}>
+        <div className={styles.clipTextWithEllipsis}>
+          <OrderActionLabel orderBasketItem={orderBasketItem} />
+          <br />
+          <span className={styles.name}>{orderBasketItem.testType?.label}</span>
+          <span className={styles.label01}>
+            {!!orderBasketItem.orderError && (
+              <>
+                <br />
+                <span className={styles.orderErrorText}>
+                  <WarningIcon size={16} />
+                  &nbsp;
+                  <span className={styles.label01}>{t('error', 'Error').toUpperCase()}</span> &nbsp;
+                  {orderBasketItem.orderError.responseBody?.error?.message ?? orderBasketItem.orderError.message}
```

Comment:
Might it be more useful to the user to show a more custom error rather than a stack trace from the backend here?

Reply:
Sometimes the message here is useful, where it mentions which particular field is failing